connect/config.py

```diff
@@ -20,6 +23,8 @@ def __init__(
             products=None,
             file=None
     ):
+        # type: (str, str, Union[str, List[str]], str) -> None
```

Comment:
Probably, better to use alternate syntax for multi-parameter functions, where hint comment placed right after parameter?
   api_url=None,   # type: str
   api_key=None, # type: str
...

Reply:
If you feel that's more readable, I'll change it, the version you propose is perfectly fine and PyCharm understands it. I just used the other one because that's how PyCharm autogenerates the hints when you select 'Add type hints for function'.

Reply:
Changed.